### PR TITLE
Feature/206 emgutf add

### PR DESCRIPTION
**Pull request by jacano**
[WIP]
EmguTF libs added, along with build instructions native binaries
TensorflowLite with a wip mobilenetv2 net to do inference for pet object detection

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -328,3 +326,6 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
+
+# Git
+*.orig
```
Comment (MarcosCobena) [inline .gitignore:331]: Yeah!

File: Source/TailwindTraders.Mobile/Directory.Build.props [hunk 2]
```diff
@@ -1,19 +1,20 @@
 <Project>
+
 	<PropertyGroup Condition="'$(Configuration)'=='Release'">
 		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
 		<StyleCopTreatErrorsAsWarnings>false</StyleCopTreatErrorsAsWarnings>
 	</PropertyGroup>
 	
 	<PropertyGroup>
-		<CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)\ruleset.ruleset</CodeAnalysisRuleSet>
+		<CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)\CodeAnalysis\CodeAnalysis.ruleset</CodeAnalysisRuleSet>
```
Comment (MarcosCobena) [inline Source/TailwindTraders.Mobile/Directory.Build.props:9]: Fantastic :-)

File: Source/TailwindTraders.Mobile/TailwindTraders.Mobile.Android/Features/Scanning/Photo/PlatformService.cs [hunk 3]
```diff
@@ -47,6 +47,78 @@ public Task<bool> ResizeImageAsync(string filePath, PhotoSize photoSize, int qua
             }
         }
 
+        public string GetContent(string v)
```
Comment (MarcosCobena) [inline Source/TailwindTraders.Mobile/TailwindTraders.Mobile.Android/Features/Scanning/Photo/PlatformService.cs]: Could we rename 'v' to something more valuable? In general try to do this by default

File: Source/TailwindTraders.Mobile/TailwindTraders.Mobile.Android/TailwindTraders.Mobile.Android.csproj [hunk 4]
```diff
@@ -58,24 +58,24 @@
...
     </PackageReference>
     <PackageReference Include="Xam.Plugin.Media">
       <Version>4.0.1.5</Version>
     </PackageReference>
-    <PackageReference Include="Xamarin.Android.Support.Design" Version="27.0.2.1" />
-    <PackageReference Include="Xamarin.Android.Support.v7.AppCompat" Version="27.0.2.1" />
-    <PackageReference Include="Xamarin.Android.Support.v4" Version="27.0.2.1" />
-    <PackageReference Include="Xamarin.Android.Support.v7.CardView" Version="27.0.2.1" />
-    <PackageReference Include="Xamarin.Android.Support.v7.MediaRouter" Version="27.0.2.1" />
+    <PackageReference Include="Xamarin.Android.Support.Design" Version="28.0.0-preview8" />
```
Comment (MarcosCobena) [inline Source/TailwindTraders.Mobile/TailwindTraders.Mobile.Android/TailwindTraders.Mobile.Android.csproj:69]: Mmm I don't like too much relying in preview packages, but let's discuss whether such makes sense here

File: Source/TailwindTraders.Mobile/TailwindTraders.Mobile/Features/Common/TensorflowLite.cs [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+        public static void DoNotStripMe()
+        {
+        }
+
+        public static void Recognize()
+        {
+            Initialize();
+
+            var imagePath = platformService.CopyToFilesAndGetPath(ImageFilename);
+            Console.WriteLine($"Using image: {imagePath}");
```
Comment (MarcosCobena) [inline Source/TailwindTraders.Mobile/TailwindTraders.Mobile/Features/Common/TensorflowLite.cs]: ILoggingService, just if this' not a WIP

Comment (jacano) [reply Source/TailwindTraders.Mobile/TailwindTraders.Mobile/Features/Common/TensorflowLite.cs]: Good catch! I do

File: Source/TailwindTraders.Mobile/TailwindTraders.Mobile.Android/TailwindTraders.Mobile.Android.csproj (hunk 4 above)
Comment (jacano) [reply Source/TailwindTraders.Mobile/TailwindTraders.Mobile.Android/TailwindTraders.Mobile.Android.csproj:69]: I think the same. What happened here is that I nuget bump SkiaSharp.Views.Forms, and this required me to update the target framework of the Android project to Android Pie (28), to stay out of warnings. But unfortunately the support libraries associated to this version are in preview. I have used the app and is working, so we can work with them the same way we are early-adopting Xamarin Forms 4.  What do you think?

File: Source/TailwindTraders.Mobile/TailwindTraders.Mobile.Android/Features/Scanning/Photo/PlatformService.cs (hunk 3 above)
Comment (jacano) [reply Source/TailwindTraders.Mobile/TailwindTraders.Mobile.Android/Features/Scanning/Photo/PlatformService.cs]: Good catch! I do

File: Source/TailwindTraders.Mobile/TailwindTraders.Mobile.Android/TailwindTraders.Mobile.Android.csproj (hunk 4 above)
Comment (MarcosCobena) [reply Source/TailwindTraders.Mobile/TailwindTraders.Mobile.Android/TailwindTraders.Mobile.Android.csproj:69]: OK!

**Review by MarcosCobena — APPROVED** [review]
Thanks for making those changes :-)

**Pull request merged**